### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by dragosiordachioaia**
It has a few more changes to the previous stuff to improve the styling, and obviously the blog part
This is a DRAFT (forgot to make it one when I opened it)

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import React from "react";
 import "./App.scss";
+// import "../MyFontsWebfontsKit.css";
```
Comment (mec07) [inline src/App/App.js]: remove this line

File: src/App/App.scss [hunk 2]
```diff
@@ -2,38 +2,88 @@
...
-  font-weight: 700;
+  font-weight: 600;
+  letter-spacing: -1px;
+}
+
+h5 {
+  text-transform: uppercase;
+  color: $darkGray;
+  font-size: 0.9rem;
+  // font-weight: bold;
```
Comment (mec07) [inline src/App/App.scss]: shall we remove this?

File: src/Blog/Blog.jsx [hunk 3]
```diff
@@ -1,11 +1,78 @@
...
+  }
+
+  return (
+    <div className="page-blog">
+      <BoxedLayout>
+        <PageTitle value="Blog" />
+        <Row className="content">
+          <Col md={6}>
+            <div className="cover-container">
+              <img src={TechImage} alt="Open laptop" />
```
Comment (mec07) [inline src/Blog/Blog.jsx]: This alt description is a little out of date. Perhaps don't describe the image, instead describe what the image stands for, e.g. "Tech blog posts icon"

File: src/Blog/Blog.jsx [hunk 4]
```diff
@@ -1,11 +1,78 @@
...
+            </div>
+            <h5 className="section-title">Tech</h5>
+            {displayBlogPosts("tech")}
+          </Col>
+          <Col md={6}>
+            <div className="cover-container">
+              <img src={IdeasImage} alt="Sliced lemon on blue background" />
```
Comment (mec07) [inline src/Blog/Blog.jsx]: Similar comment to the above

File: src/Blog/Blog.scss [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+    p {
+      font-size: 1.1rem;
+    }
+
+    .meta-container {
+      margin-top: -1.5rem;
+
+      p {
+        // font-family: Poppins, Arial;
```
Comment (mec07) [inline src/Blog/Blog.scss]: Remove this comment

File: src/Blog/Blog.scss [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+      .date p {
+        color: $color4;
+      }
+
+      .date {
+        text-align: right;
+      }
+    }
+
+    &:hover {
```
Comment (mec07) [inline src/Blog/Blog.scss]: shall we do the @media(hover) thing you discovered today?

File: src/Blog/blogData.json [hunk 7]
```diff
@@ -0,0 +1,21 @@
+{
+  "tech": [
+    {
+      "title": "Keeping apps in sync",
+      "abstract": "At first glance, the following situation should be easy to achieve: given a few devices running an app which displays a timer, make sure that they stay synchronised within less than 100 milliseconds.",
+      "author": "Dragos Iordachioaia",
+      "photo": "",
```
Comment (mec07) [inline src/Blog/blogData.json]: we're not using this `photo` field, so shall we just remove it?

**Comment by dragosiordachioaia** [comment]
I merged this manually, due to an issue with git.

**Pull request closed without merging**